packages/components/src/internal/app/constants.ts

```diff
@@ -90,13 +91,23 @@ export const EXPERIMENTAL_REQUESTS_MENU = 'experimental-biologics-requests-menu'
 export const EXPERIMENTAL_SAMPLE_ALIQUOT_SELECTOR = 'experimental-sample-aliquot-selector';
 export const EXPERIMENTAL_GRID_LOCK_LEFT_COLUMN = 'experimental-grid-lock-left-column';
 
+// The enum values here should align with the ProductFeature.java enum values (some not currently used but included for completeness)
+export enum ProductFeature {
+    Assay = 'Assay',
+    ELN = 'ELN',
+    FreezerManagement = 'FreezerManagement',
+    SampleManagement = 'SampleManagement',
```

Comment:
Workflow is the only feature that can't be done with a module check, since it's a professional feature but provided by the sampleManagement module.  These "FreezerManagement" and "SampleManagement" values are not really used here, just there for completeness compared to the server-side enum. And on the server side, these values are really only for documentation purposes. If it causes confusion they can be removed.